anthos-baremetal-edge-deployment/roles/ready-ubuntu/tasks/setup-vlan-interfaces.yml

```diff
@@ -0,0 +1,46 @@
+- name: Print network interfaces
```

Comment:
You can take this file out, and the reference to it in the main.yaml of the same level...this just sets up VLANs which should be a phase 2 or 3

Reply:
Removed in 40d2707. Please confirm it is correct!